core/blockchain.go

```diff
@@ -215,8 +215,8 @@ type BlockChain struct {
 
 	wg            sync.WaitGroup //
 	quit          chan struct{}  // shutdown signal, closed in Stop.
-	running       int32          // 0 if chain is running, 1 when stopped
-	procInterrupt int32          // interrupt signaler for block processing
+	stopping      atomic.Bool    // false if chain is running, true when stopped
```

Comment:
perhaps `stopped`?

Reply:
I think `stopping` is more correct -- it becomes `true` when `stopWithoutSaving` is invoked -- flipping this bool is basically the very first thing that happens on a call to `Stop()`. After this, all the waits happen. 

So this boolean does not mean it is `stopped`, but rather that the process of `stopping` the blockchain has (at least) begun. 

So IMO it's good the way it is

Reply:
Does the comment need to change to `true when stopping`?